Name_quiz.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <title>Title</title>
+    <link rel="stylesheet" href="css/all.min.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script src="https://code.iconify.design/1/1.0.7/iconify.min.js"></script>
+</head>
+<body>
+    <header>
+        <div>
+            <h1>KsisQuiz</h1>
+            <div>
+                <h1>Kim:</h1>
+                <a href="#"><i class="fas fa-home"></i></a>
+                <a href="#"><i class="far fa-user-circle"></i></a>
+            </div>
+        </div>
+    </header>
+    <div class="question-card-box">
+        <form action="handler.php" class="form-box">
+            <div class="question-card-open">
+                <div class="question-open">
+                    <div><h2>Name your quiz:<br></h2></div>
+                </div>
+                <div class="answers">
+                    <div>
+                        <input class="input" id="answer1" name="answer1"><Br><Br>
+                    </div>
+
+                </div>
+            </div>
+            <div>
+                <h2>Next:<br></h2>
+            </div>
+            <div class="button-box">
+                <button type="submit"><h2>Q. Answer</h2></button>
```

Comment:
So you omitted these things during the design stage... And as a result you got them wrong.

You should not place headers inside buttons.